packages/node/src/serverless-functions/dynamic-import.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  dynamicImport: filepath => import(filepath).then(mod => mod.default),
```

Comment:
Now you mention that, is it possible this code is taking care of that behavior?
https://github.com/vercel/vercel/pull/9745/files#diff-0b5ebdcaf8e4e9cd4e257835ef35598d820da612d518fac10122e7ab6722c9cdL75

Ideally, I want to add a some tests there. What could be an input user code to reproduce that behavior?

BTW, I added this helper just because the TypeScript configuration is overriding import to be equivalent to require

Reply:
Yes the `.default` loop in `getListener()` is designed to handle those cases